tests/common/config_reload.py

```diff
@@ -43,16 +43,6 @@ def config_system_checks_passed(duthost):
         if int(out['stdout'].strip()) < 120:
             return False
 
-    logging.info("Checking if delayed services are up")
```

Comment:
>delayed [](http://example.com/codeflow?start=30&length=7)

Do you have new ways to check delayed services?

Reply:
@qiluo-msft No. But the waiting for swss for 120 seconds should cover this as well.

Reply:
Originally the test verify lldp/pmon/snmp/telemetry/etc started. I do not understand "waiting for swss for 120 seconds should cover this".

Reply:
Previously all these services took 3 min 30 seconds to boot up and hence it is greater than 120 seconds wait for swss. Now these services almost within 30 seconds of switch initialization (at the worst a min). So essentially config reload will not proceed for another 60 seconds which is quite sufficient for these services to start.
Hence not adding any checks here.